src/authn/authn.ts

```diff
@@ -208,7 +208,7 @@ export async function logInLoadPreferences (context: AuthenticationContext): Pro
   } catch (err) {
     let m2: string
     if (err instanceof UnauthorizedError) {
-      m2 = 'Strange - you are not authenticated (properly logged in) to read preference file.'
+      m2 = 'Ooops - you are not authenticated (properly logged in) to for me to read your preference file.  Try loggin out and logging in?'
```

Comment:
```suggestion
      m2 = 'Ooops - you are not authenticated (properly logged in) for me to read your preference file.  Try logging out and logging back in?'
```

Reply:
@TallTed These files shouldn't be in this PR.  I made a mistake.  I'm going to have to take them out.  This one in particular I think it's an old version ...  I'm going to double check this, but unfortunately I may lose your comments.  I'm really sorry about this.  After doing it I will double check all solid-ui files for typos and make another PR.